content/yaml-running-builds/starting-builds-automatically.md

```diff
@@ -65,6 +68,17 @@ triggering:
   cancel_previous_builds: false  # Set to `true` to automatically cancel outdated webhook builds
 {{< /highlight >}}
 
+### Pattern examples
+
+| Pattern | Explanation |
+|-|-|
+|`*`| Matches everything |
+|`*-dev`| Values with suffix `-dev`, e.g. `v0.0.42-dev`|
+|`!(*-dev)`| Values without suffix `-dev`, e.g. `v0.0.42`|
+|`{test,qa}/*`| Values with prefix `test/` and `qa/`, e.g. `test/popup`|
```

Comment:
For consistency and better readability, I'd recommend starting the explanation with a verb, e.g. "Matches values with the suffix `-dev`."